genesyscloud/workforcemanagement_businessunits/resource_genesyscloud_workforcemanagement_businessunits_schema.go

```diff
@@ -0,0 +1,226 @@
+package workforcemanagement_businessunits
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/mypurecloud/terraform-provider-genesyscloud/genesyscloud/provider"
+	resourceExporter "github.com/mypurecloud/terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	registrar "github.com/mypurecloud/terraform-provider-genesyscloud/genesyscloud/resource_register"
+)
+
+/*
+ResourceName is defined in this file along with four functions:
+
+1.  The registration code that registers the Datasource, Resource and Exporter for the package.
+2.  The resource schema definitions for the workforcemanagement_businessunits resource.
+3.  The datasource schema definitions for the workforcemanagement_businessunits datasource.
+4.  The resource exporter configuration for the workforcemanagement_businessunits exporter.
+*/
+const ResourceName = "genesyscloud_workforcemanagement_businessunits"
+
+// SetRegistrar registers all the resources, datasources and exporters in the package
+func SetRegistrar(regInstance registrar.Registrar) {
+	regInstance.RegisterResource(ResourceName, ResourceWorkforcemanagementBusinessunits())
+	regInstance.RegisterDataSource(ResourceName, DataSourceWorkforcemanagementBusinessunits())
+	regInstance.RegisterExporter(ResourceName, WorkforcemanagementBusinessunitsExporter())
+}
+
+// ResourceWorkforcemanagementBusinessunits registers the genesyscloud_workforcemanagement_businessunits resource with Terraform
+func ResourceWorkforcemanagementBusinessunits() *schema.Resource {
+	buShortTermForecastingSettingsResource := &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			`default_history_weeks`: {
+				Description: `The number of historical weeks to consider when creating a forecast. This setting is only used for legacy weighted average forecasts`,
+				Optional:    true,
+				Type:        schema.TypeInt,
+			},
+		},
+	}
+
+	schedulerMessageTypeSeverityResource := &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			// See API documentation for valid enum values: https://developer.mypurecloud.com/api/rest/v2/workforcemanagement/
+			`type`: {
+				Description: `The type of the message. Validation is handled by the API to avoid maintaining a potentially stale list of enum values. See API documentation for valid values: https://developer.mypurecloud.com/api/rest/v2/workforcemanagement/`,
```

Comment:
Update link to https://developer.genesys.cloud/useragentman/workforcemanagement/#post-api-v2-workforcemanagement-businessunits